File: cpp_solutions/2390. Removing Stars From a String.cc
class Solution {
public:
    string removeStars(string s) {
        stack<char> stacks;
        for(char it : s) {
            if (string(1,it) != "*" ) {
                stacks.push(it);
            } else {
                while(string(1,stacks.top()) =="*") {
                    stacks.pop();
                }
                stacks.pop();
            }
        }
        
        string ans = "";
        while (!stacks.empty()) {
            ans+=stacks.top();
            stacks.pop();
        }
        reverse(ans.begin(),ans.end());
        return ans;
    }
};

File: python3_solution_set2/2390. Removing Stars From a String.py
class Solution:
    def removeStars(self, s: str) -> str:
        stack = []
        
        for x in s:
            if x!="*":
                stack.append(x)
            else:
                while stack and stack[-1]=="*":
                    print("inside")
                    
                stack.pop()
        
        return "".join(stack)
        
